practitioner-service/src/test/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+# parameterise this!
+fhir.uri=http://fhir-api:8080
+server.error.include-message=always
```

Comment:
Can you explain why this file is required?

Reply:
This file is no longer needed, removed it